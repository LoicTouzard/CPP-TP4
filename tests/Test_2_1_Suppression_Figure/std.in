C c 0 0 1
DELETE c
LIST
EXIT
