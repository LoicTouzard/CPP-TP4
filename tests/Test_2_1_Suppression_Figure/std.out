OK
#New object :c
OK
#The list is empty
